homeassistant/components/google_assistant/trait.py

```diff
@@ -1670,7 +1671,7 @@ async def execute(self, command, data, params, challenge):
                     ATTR_ENTITY_ID: self.state.entity_id,
                     media_player.ATTR_SOUND_MODE: sound_mode,
                 },
-                blocking=True,
+                blocking=not self.config.should_report_state,
```

Comment:
Are these changes still necessary if we don't wait up to 2 seconds anyway?

Reply:
I think it's nice to return immediately if we're anyway pushing updates back to Google, there's no point in waiting?